library_generation/utils/utilities.sh

```diff
@@ -282,7 +315,7 @@ get_proto_path_from_preprocessed_sources() {
   pushd "${sources}" > /dev/null
   local proto_library=$(find . -maxdepth 1 -type d -name 'proto-*' | sed 's/\.\///')
   local found_libraries=$(echo "${proto_library}" | wc -l)
-  if [ -z ${proto_library} ]; then
+  if [[ -z ${proto_library} ]]; then
```

Comment:
I noticed that the shell unit tests were expecting the binary brackets because `proto_library` could be expanded to a string with spaces.

```
+ '[' -z proto-2 proto-1 ']'
/usr/local/google/home/diegomarquezp/google/sdk-platform-java/library_generation/test/../utils/utilities.sh: line 318: [: proto-2: binary operator expected
```